File: kaftter/src/main/java/kaftter/repository/TweetRepository.java
package kaftter.repository;

import kaftter.domain.TweetEntity;
import kaftter.domain.TweetKey;
import org.springframework.data.cassandra.repository.CassandraRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface TweetRepository extends CassandraRepository<TweetEntity, TweetKey> {}

File: kaftter-api/src/main/kotlin/kaftter/api/repository/TweetRepository.kt
package kaftter.api.repository

import kaftter.api.domain.TweetEntity
import kaftter.api.domain.TweetKey
import org.springframework.data.cassandra.repository.CassandraRepository
import org.springframework.stereotype.Repository

@Repository
interface TweetRepository : CassandraRepository<TweetEntity, TweetKey>